src/content/en/updates/2018/09/inside-browser-part1.md

```diff
@@ -0,0 +1,298 @@
+
+When you start an application, a process is created. The program might create thread(s) to help it 
+do work, but that's optional. The Operating System gives the process a "slab" of memory to work 
+with and all application state is kept in that private memory space. When you close the 
+application, the process also goes away and the Operating System frees up the memory.
+<div class="clearfix"></div>
+
+<figure>
+  <img src="../../images/inside-browser/memory.svg" alt="process and memory">
+  <figcaption>
+   Figure 5: Diagram of a process using memory space and storing application data
+  </figcaption>
+</figure>
+
+A process can ask the Operating System to spin up another process to run different tasks. When this 
+happens, different parts of the memory are allocated for the new process. If two processes need to 
+talk, they can do so by using **I**nter **P**rocess **C**ommunication (**IPC**). Many applications 
+are designed to work this way so that if a worker process get unresponsive, it can be restarted 
+without stopping other processes which are running different parts of the application.
+
+<figure>
+  <img src="../../images/inside-browser/workerprocess.svg" alt="worker process and IPC">
+  <figcaption>
+   Figure 6: Diagram of separate processes communicating over IPC
+  </figcaption>
+</figure>
+
+
+## Browser Architecture {: #browser-architecture }
+
+So how is a web browser built using processes and threads? Well, it could be one process with many 
+different threads or many different processes with a few threads communicating over IPC. 
+<figure>
+  <img src="../../images/inside-browser/browser-arch.png" alt="browser architecture">
+  <figcaption>
+   Figure 7: Different browser architectures in process/thread diagram
+  </figcaption>
+</figure>
+
+The important thing to note here is that these different architectures are implementation details. 
+There is no standard specification on how one might build a web browser. One browser’s approach may 
+be completely different from another. 
+
+For the sake of this blog series, we are going to use Chrome’s recent architecture described in the 
+diagram below.
+
+At the top is the browser process coordinating with other processes that take care of different 
+parts of the application. For the renderer process, multiple processes are created and assigned to 
+each tab. Until very recently, Chrome gave each tab a process when it could; now it tries to give 
+each site its own process, including iframes (see [Site Isolation](#site-isolation)).
+
+<figure>
+  <img src="../../images/inside-browser/browser-arch2.png" alt="browser architecture">
+  <figcaption>
+   Figure 8: Diagram of Chrome’s multi-process architecture. Multiple layers are shown under 
+   Renderer Process to represent Chrome running multiple Renderer Processes for each tab.
+  </figcaption>
+</figure>
+
+## Which process controls what?
+The following table describes each Chrome process and what it controls:
+
+<table>
```

Comment:
Nit - Consider making the table [responsive](https://developers.google.com/internal/publishing/reference/styles#responsive_tables)